src/mudlet-lua/lua/Other.lua

```diff
@@ -3,7 +3,7 @@
 ----------------------------------------------------------------------------------
 
 mudlet = mudlet or {}
-mudlet.supportscoroutines
+mudlet.supportscoroutines = true
```

Comment:
Well if you look at the first commit then you'll see that it was exactly that! However, as I mentioned in the PR, older Mudlets will error on the code of `mudlet.supports.coroutines` if you use it in a package that then gets imported by a player running Mudlet 2.x, for example - since `mudlet.supports` resolves to `nil` and `nil` doesn't have `coroutines`.